src/voting/templatetags/voting_tags.py

```diff
@@ -64,8 +64,8 @@ def __init__(self, user, objects, context_var):
 
     def render(self, context):
         try:
-            user = template.resolve_variable(self.user, context)
-            objects = template.resolve_variable(self.objects, context)
+            user = template.Variable(self.user).resolve(context)
+            objects = template.Variable(self.object).resolve(context)
```

Comment:
It's probably just a typo, all the other ones are correct but for this one you forgot to add the s when you changed this code.